KMeans.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.cluster import KMeans
+from sklearn.datasets import make_blobs
+
+'''
```

Comment:
please refer to the style guide for the conventions we are hoping to follow (https://numpydoc.readthedocs.io/en/latest/format.html#short-summary). block comments below def statement